.eslintrc

```diff
@@ -72,7 +72,7 @@
     "babel/generator-star-spacing": ["error", { before: false, after: true }],
 
     "react/display-name": ["error", { ignoreTranspilerName: true }],
-
+    "react/prefer-stateless-function": ["off"],
```

Comment:
https://medium.com/modus-create-front-end-development/component-rendering-performance-in-react-df859b474adc#.u5375qyp7